File: app/src/main/java/de/christinecoenen/code/zapp/models/shows/Quality.java
package de.christinecoenen.code.zapp.models.shows;

import de.christinecoenen.code.zapp.R;

public enum Quality {
	Low(R.string.fragment_mediathek_qualities_low),
	Medium(R.string.fragment_mediathek_qualities_medium),
	High(R.string.fragment_mediathek_qualities_high);

	private final int labelResId;

	Quality(int labelResId) {
		this.labelResId = labelResId;
	}

	public int getLabelResId() {
		return labelResId;
	}
}

File: app/src/main/java/de/christinecoenen/code/zapp/models/shows/Quality.kt
package de.christinecoenen.code.zapp.models.shows

import androidx.annotation.StringRes
import de.christinecoenen.code.zapp.R

enum class Quality(@StringRes val labelResId: Int) {

	Low(R.string.fragment_mediathek_qualities_low),
	Medium(R.string.fragment_mediathek_qualities_medium),
	High(R.string.fragment_mediathek_qualities_high);

}
